filebeat/module/system/syslog/test/suse-syslog.log-expected.json

```diff
@@ -0,0 +1,28 @@
+[
+    {
+        "ecs.version": "1.0.0-beta2",
```

Comment:
I don't see an `@timestamp` in the output here. Do I miss something?

Reply:
@ruflin unfortunately all the timestamps got removed in https://github.com/elastic/beats/pull/10736/files/5c8b9d3205efc146891117823f724935600ed372#diff-7f560f729ae36bb40472cb039fedc34fL42

Reply:
Sorry, regarding to the `@timestamp` field: https://github.com/elastic/beats/blob/master/filebeat/tests/system/test_modules.py#L220 because timestamp got removed from system.syslog metricset

Reply:
I see. I now understand why it happens. The problem now is that the log file above actually has timestamps with the year inside but we don't test it. This becomes especially important as we just added a pattern to support year but the tests we have to confirm it works, don't do that.

Is there an easy way to fix this?